### PR TITLE
Fix various docs format

### PR DESCRIPTION
**Pull request by fehiepsi**
As title.

### CODE REVIEW

File: pyro/infer/importance.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 import torch
 import warnings
 
-from pyro.distributions.util import logsumexp
```
Comment (fehiepsi) [inline pyro/infer/importance.py]: This leads to a wrong display for logsumexp http://docs.pyro.ai/en/0.3.0-release/inference_algos.html#pyro.infer.importance.logsumexp

File: pyro/primitives.py [hunk 2]
```diff
@@ -273,9 +273,9 @@ def module(name, nn_module, update_module_params=False):
 
 
 def random_module(name, nn_module, prior, *args, **kwargs):
-    """
+    r"""
     Places a prior over the parameters of the module `nn_module`.
-    Returns a distribution (callable) over `nn.Module`s, which
+    Returns a distribution (callable) over `nn.Module`\s, which
```
Comment (fehiepsi) [inline pyro/primitives.py]: This fixes the format at http://docs.pyro.ai/en/0.3.0-release/primitives.html#pyro.random_module. I don't know a better solution.

File: docs/source/ops.rst [hunk 3]
```diff
@@ -38,3 +44,12 @@ Tensor Contraction
...
 .. autofunction:: pyro.ops.contract.ubersum
+
+Statistical Utilities
+---------------------
+
+.. automodule:: pyro.ops.stats
+    :members:
+    :undoc-members:
+    :show-inheritance:
+    :member-order: bysource
```
Comment (fehiepsi) [inline docs/source/ops.rst]: Docs for stats are missing. There aren't docs for many other ops too but it might be an intention so I don't touch them.

**Comment by fritzo** [comment]
Thanks for cleaning up docs, @fehiepsi !

**Pull request merged**